apps/vms_term_sock.c

```diff
@@ -317,7 +317,7 @@ static int CreateSocketPair (int SocketFamily,
     /*
     ** Initialize the socket information
     */
-    slen = sizeof (sin);
+    slen = sizeof(sin);
     memset ((char *) &sin, 0, slen);
```

Comment:
Same comment as in #4876, but three occurrences here...

Reply:
If cross-compared to master, it might be more appropriate to dismiss this comment, even in #4876. I mean it's probably more appropriate to address it separately [if at all].